conans/client/toolchain/meson.py

```diff
@@ -0,0 +1,176 @@
+import os
+
+from conans.client.build.cppstd_flags import cppstd_from_settings
+from conans.util.files import save
+
+import textwrap
+from jinja2 import Template
+
+
+class MesonToolchain(object):
+    _native_filename = "conan_meson_native.ini"
+    _cross_filename = "conan_meson_cross.ini"
+
+    _native_file_template = textwrap.dedent("""
+[project options]
```

Comment:
Great thing of dedent is that you can actually indent the  contents, so the python file reads well indented. The resulting template will not be indented.

Reply:
thanks, indented